spec/tags/core/kernel/block_given_tags.txt

```diff
@@ -1,3 +1,3 @@
 fails:Kernel#block_given? returns false when a method defined by define_method is called with a block
 fails:Kernel.block_given? returns false when a method defined by define_method is called with a block
-fails:self.send(:block_given?) returns true if and only if a block is supplied
+fails:self.send(:block_given?) returns false when a method defined by define_method is called with a block
```

Comment:
I wouldn't worry too much about this regression. If you're confident you're doing the right thing it's just one spec and it may not have been right in the first place.